sym-files2/databasebackend.py

```diff
@@ -166,38 +165,50 @@ def _process_data(self):
             'xlabel_addition': '',
             'y_left_label_addition': '',
             'y_right_label_addition': '',
-            } 
+            }
         # Don't do data processing on date data
         if self.ggs['default_xscale'] == 'dat':
             return
-        # Run plugin(s) on data
-        self._plugins()
+
         # Convert the data to be as a function of ...
         if self.o['as_function_of']:
             self._as_function_of()
+
+        # Run plugin(s) on data
```

Comment:
Are you sure this is what we want. To convert to as function of, before running plugins. I don't remember the design and use cases well enough to be sure

Reply:
I am as sure as I think I am going to be. We thought about the various cases, an as far as I can see, this is generally the wanted behavior.